liblepton/scheme/lepton/object.scm

```diff
@@ -33,13 +33,15 @@
 
   #:use-module (lepton color-map)
   #:use-module (lepton ffi)
+  #:use-module (lepton object foreign)
   #:use-module (lepton object type)
 
```

Comment:
I think now you can safely remove
`#:use-module (lepton core object)`

Reply:
Yes, I'll do it in the next patchset.